pages/common/pdfjoin.md

```diff
@@ -1,15 +1,20 @@
 # pdfjoin
 
-> PDF merging utility.
+> PDF merging utility based on pdfjam.
+> More information: <https://github.com/rrthomas/pdfjam-extras>.
 
-- Merge two PDFs:
+- Merge two PDFs into one with the default suffix "joined":
 
-`pdfjoin {{file1}} {{file2}} --outfile {{output_file}}`
+`pdfjoin {{file1}} {{file2}}`
+
+- Merge the first page of each given file together:
+
+`pdfjoin {{files...}} 1 --outfile {{output_file}}`
```

Comment:
Is this supposed to be an arbitrary length list of files? If so, I think this would be more clear:
```suggestion
`pdfjoin {{file1 file2 ..}} 1 --outfile {{output_file}}`
```

Reply:
Generally speaking the convention in other pages other pages is like this:
```suggestion
`pdfjoin {{file1}} {{file2}} {{fileN}} 1 --outfile {{output_file}}`
```